components/base/kinematicbase/kinematics.go

```diff
@@ -93,6 +101,13 @@ type Options struct {
 	// PositionOnlyMode defines whether motion planning should be done in 2DOF or 3DOF.
 	// If value is true, planning is done in [x,y]. If value is false, planning is done in [x,y,theta].
 	PositionOnlyMode bool
+
+	// UsePTGs defines whether motion planning should plan using PTGs.
+	UsePTGs bool
+
+	// SkidSteer defines whether motion planning should plan for diff drive bases using skid steer. If true, it will plan using
+	// only rotations and straight lines. Not used if turning radius > 0, or if UsePTGs is false.
+	NoSkidSteer bool
```

Comment:
I would rather not introduce this now and have the default behavior be we only use move straight and spin PTGs for when R=0

Reply:
That's not the desired behavior, though, as I understand it; the reason we're doing this is that a Viam rover should be able to turn a corner without stopping.

Reply:
Earlier we agreed that for the moment it was sufficient to preserve current behavior but switch to using set velocity calls (ie. PTGs).  If a user wants to use a viam rover to turn without stopping they can still configure it using a nonzero R.  In the future we might want to tackle to problem of a rover that can turn in place and also have some turning radius from skid steering but that is out of scope for this ticket.

Reply:
Ok, will update default to disable skid steering

Reply:
@tessavitabile